examples/plots/plot_interpolation_for_transform_manager.py

```diff
@@ -0,0 +1,93 @@
+"""
+==================================
+Managing Transformations over Time
+==================================
+
+In this example, given two trajectories of 3D rigid transformations, we will
+interpolate both and use the transform manager for the target timestep.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pytransform3d import rotations as pr
+from pytransform3d import transformations as pt
+from pytransform3d.transform_manager import (TemporalTransformManager,
+                                             NumpyTimeseriesTransform)
+
+
+def create_sinusoidal_movement(
```

Comment:
Consider moving it to a utility module (which is excluded from the python wheel) but still can be usable in example plots or documentation - here we have a real duplication of things...